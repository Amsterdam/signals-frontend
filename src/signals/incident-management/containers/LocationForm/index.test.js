// import React from 'react';
// import { shallow } from 'enzyme';

// import { LocationForm } from 'index';

describe('<LocationForm />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(true);
  });
});
