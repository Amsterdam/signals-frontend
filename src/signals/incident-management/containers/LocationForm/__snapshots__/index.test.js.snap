// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationForm /> should render correctly 1`] = `
<div
  className="incident-location-container"
>
  <Component
    incidentModel={
      Object {
        "incident": Object {
          "id": 42,
          "location": Object {
            "foo": "bar",
          },
        },
        "patching": Object {
          "location": false,
        },
      }
    }
    onPatchIncident={[MockFunction]}
  />
</div>
`;
